source/clear-linux/get-started/live-image.rst

```diff
@@ -26,6 +26,10 @@ Boot the Clear Linux live image
 
 #. Plug the imaged USB drive in and boot it up.
 #. Log in as `root` and set a password.
+   
+To start configuring the new environment, click `create and enable user space`_. 
+
+.. _create and enable user space: https://clearlinux.org/documentation/clear-linux/guides/maintenance/enable-user-space 
```

Comment:
I don't think this addition (lines 30-32) is necessary.

Reply:
Omitted. Done.